app/jobs/zipmaker_job.rb

```diff
@@ -9,6 +10,26 @@ class ZipmakerJob < ApplicationJob
   # @param [String] druid
   # @param [Integer] version
   def perform(druid, version)
+    binary_path = Moab::StorageServices.object_version_path(druid, version)
```

Comment:
why "binary"?  what do you think of "moab_version_path" ?

(binary appears in a lot of places here ... )

Reply:
Also:  instead of passing the moab version path and zip path around .. should they be attributes?   Memoized methods, with druid and version being the attributes?

Because I noticed you are passing the same arguments to 2 methods.

Reply:
It is markedly harder to do attributes on jobs, since you don't get to control their initialization.  I think what you are suggesting is total overkill.